dkg/dkg_test.go

```diff
@@ -405,3 +408,134 @@ func startNewDKG(t *testing.T, parentCtx context.Context, config dkg.Config, dkg
 
 	return cancel
 }
+
+// TestKeymanager tests the keymanager functionality where the dkg command pushes validator keyshares
+// to a keymanager API url.
+func TestKeymanager(t *testing.T) {
```

Comment:
this feels like a duplicate test, can't we add this to existing table test?

Reply:
yeah we can, doing this now